{"uuid": "cf9e3bb3-0147-468b-985f-fae0c030ce16", "children": ["c7cc307e-6a1a-4730-b28b-dc1b6284cc18", "55247896-9545-4b6c-85b9-0a2af88d89fe", "a4902548-4b0a-4d96-8fde-bb77112e17ae", "7085a679-8ee9-4a46-8adf-ae13cf4c1329", "ad58c78b-138f-4e64-8bce-9aa49f57d1da", "8ae80ab8-e5ac-432e-a9d9-13f4b05c53eb", "9ec0871f-1b97-4898-9735-99d3646162b3", "844383bb-713f-4572-8e18-800255fe08c5", "c3b5ca09-6af5-4777-947e-9c9d263ec35a", "a6fd4090-7d92-4559-95fa-d55371a6e2d0", "81873b0b-f147-4f75-85fc-6636d15da656", "7a90bc45-69d5-4d3c-9527-1049791c2caa"], "befores": [{"name": "ensure_user_created", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n  +403\n  -200\n", "trace": "  File \"c:\\users\\иван\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\иван\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\иван\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Иван\\Desktop\\Diplom\\Diplom_2\\tests\\conftest.py\", line 26, in ensure_user_created\n    assert response.status_code == 200\n"}, "start": 1744991681645, "stop": 1744991682795}], "start": 1744991681645, "stop": 1744991683704}